airflow/cli/cli_parser.py

```diff
@@ -1054,6 +1080,14 @@ class GroupCommand(NamedTuple):
         args=(ARG_CLEAR_ONLY,),
     ),
 )
+MAINTENANCE_COMMANDS = (
+    ActionCommand(
+        name='cleanup',
```

Comment:
```suggestion
        name='cleanup-tables',
```

Reply:
In the future, we can add other commands that will clean other things, e.g. logs.